presto-main/src/test/java/com/facebook/presto/sql/analyzer/TestMaterializedViewQueryOptimizer.java

```diff
@@ -373,28 +391,28 @@ public void testWithTableAlias()
-        assertOptimizedQuery(originalViewSql, baseQuerySqlWithAliasPartially2, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSql, baseQuerySqlFully, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSql, baseQuerySqlWithTablePrefix, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasPartially, baseQuerySql, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasPartially, baseQuerySqlWithAliasPartially1, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasPartially, baseQuerySqlWithAliasPartially2, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasPartially, baseQuerySqlFully, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasPartially, baseQuerySqlWithTablePrefix, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasFully, baseQuerySql, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasFully, baseQuerySqlWithAliasPartially1, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasFully, baseQuerySqlWithAliasPartially2, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasFully, baseQuerySqlFully, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithAliasFully, baseQuerySqlWithTablePrefix, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithTablePrefix, baseQuerySql, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithTablePrefix, baseQuerySqlWithAliasPartially1, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithTablePrefix, baseQuerySqlWithAliasPartially2, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithTablePrefix, baseQuerySqlFully, expectedRewrittenSql);
-        assertOptimizedQuery(originalViewSqlWithTablePrefix, baseQuerySqlWithTablePrefix, expectedRewrittenSql);
+        String expectedRewrittenSql = format("SELECT a, c FROM %s ORDER BY c, a", VIEW_1);
+
+        assertOptimizedQuery(baseQuerySql, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
+        assertOptimizedQuery(baseQuerySqlWithAliasPartially1, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
+        assertOptimizedQuery(baseQuerySqlWithAliasPartially2, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
+        assertOptimizedQuery(baseQuerySqlFully, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
+        assertOptimizedQuery(baseQuerySqlWithTablePrefix, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
+        assertOptimizedQuery(baseQuerySql, expectedRewrittenSql, ImmutableMap.of(BASE_TABLE_1, ImmutableList.of(originalViewSql)));
```

Comment:
It seems like the same lines are copied-paste 4 times?

Reply:
Good catch - addressing this comment: https://github.com/prestodb/presto/pull/17901#discussion_r906271443

Required using the refactor tool then some replacing, and I made the assumption that the existing tests all used "originalViewSql". My bad for not catching that in my own review.